dashboard/app/views/pd/workshop_survey/new.html.haml

```diff
@@ -0,0 +1,18 @@
+- course = @pd_enrollment.workshop.course
+- subject = @pd_enrollment.workshop.subject
+- @page_title = "Satisfaction Survey for Code.org's #{course} #{subject} Professional Development Workshop"
+
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+  %script{src: minifiable_asset_path('js/pd/workshop_survey/new.js'), data: @script_data}
```

Comment:
It's getting added in https://github.com/code-dot-org/code-dot-org/pull/15037, but it does seem weird to not have it in this PR when it's referenced. But at the same time, the view needs to be here for the route to make sense.

Perhaps I'll add a relatively empty version of the js file here?